apps/src/templates/teacherDashboard/SectionScriptProgress.story.js

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import SectionScriptProgress from './SectionScriptProgress';
+import { TestResults } from '@cdo/apps/constants';
+
+// A truncated copy of the scriptData for csp1
+const scriptData = {
+  "id": 112,
+  "name": "csp1",
+  "stages": [
+    {
+      "script_id": 112,
+      "script_name": "csp1",
+      "script_stages": 17,
+      "freeplay_links": [],
+      "id": 1360,
+      "position": 1,
+      "name": "CS Principles Pre-survey",
+      "title": "CS Principles Pre-survey",
+      "flex_category": "Content",
+      "lockable": true,
+      "levels": [
+        {
+          "ids": [
+            8986
+          ],
+          "activeId": 8986,
+          "position": 1,
+          "kind": "assessment",
+          "icon": "fa-check-square-o",
+          "is_concept_level": false,
+          "title": 1,
+          "url": "http://localhost-studio.code.org:3000/s/csp1/lockable/1/puzzle/1/page/1",
```

Comment:
does it make sense to do a relative or prod url here?

Reply:
I don't think it really matters. It's not really designed for clicking through in storybook. Mostly I just wanted the story so that we have at least some test coverage.